horovod/spark/common/params.py

```diff
@@ -53,8 +53,7 @@ class EstimatorParams(Params):
     label_cols = Param(Params._dummy(), 'label_cols', 'label column names',
                        typeConverter=TypeConverters.toListString)
     validation = Param(Params._dummy(), 'validation',
-                       'one of: float validation split [0, 1), or string validation column name',
-                       typeConverter=TypeConverters.toString)
```

Comment:
why is the string converter removed?

Reply:
validation: Optional validation column name (string) where every row in the column is either 1/True or 0/False,
                    or validation split (float) giving percent of data to be randomly selected for validation.

Reply:
It can be a string for col name or float for split.

Reply:
and with that converter it always turned it into strings?